src/components/Feed/ActivityStatusChange/hooks/getContextParents.js

```diff
@@ -0,0 +1,44 @@
+import { useSelector } from "react-redux"
+
+const useGetContextParents = (activity, entityType) => {
+  const focusedFolders = useSelector((state) => state.context.focused.folders)
+  let tag = []
+  let tagTypes = {}
+
+
+  if (activity.activityData.origin.type == 'folder' && !tagTypes.folder) {
+    tagTypes.folder = true
+    tag.push(activity.activityData.origin.name)
+  }
+
+  if (focusedFolders?.length > 1 && !tagTypes.folder) {
+    tagTypes.folder = true
+    tag.push(activity.activityData.parents.find((p) => p.type == 'folder')?.name)
```

Comment:
I managed to create a type error by selecting multiple folders and then selecting a task.

![image](https://github.com/user-attachments/assets/3a735e76-a37c-4349-b4f2-101c9ccc8e82)
![image](https://github.com/user-attachments/assets/ebd4a866-e647-4ddb-a52e-6e9e8eb355eb)